addons/mail/static/tests/core/record.test.js

```diff
@@ -1048,3 +1048,47 @@ test("insert with id relation keeps existing field values", async () => {
     expect(member2.channel.eq(channel1)).toBe(true);
     expect(member2.is_internal).toBe(true);
 });
+
+test("accessing to inverse of a computed field should trigger recomputation", async () => {
+    (class Thread extends Record {
+        static id = "id";
+        id;
+        messageToCompute = Record.one("Message", {
+            compute() {
+                expect.step("compute messageToCompute");
+                return this.messages[0];
+            },
+            inverse: "threadOfComputedMessage",
+        });
+        messages = Record.many("Message");
+    }).register(localRegistry);
+
+    (class Message extends Record {
+        static id = "id";
+        id;
+        threadOfComputedMessage = Record.one("Thread", { inverse: "messageToCompute" });
+    }).register(localRegistry);
+
+    const store = await start();
+    const thread = store.Thread.insert({ id: 1 });
+
+    let renderCount = 0;
+    function render() {
+        renderCount++;
+        expect.step(`render ${renderCount}`);
+        for (const message of thread.messages) {
+            message.threadOfComputedMessage;
+        }
+    }
+
+    reactive(thread, render);
```

Comment:
- nitpick: IDE unhappy with just accessor in statement. IDE's happy if you put `void message.threadOfComputedMessage;`
- the `render()` should observe `message.threadOfComputedMessage` using the result of `reactive(thread, render)`, i.e. a reactive that observes and is automatically triggering its callback `render` from changed. This simulates component templates.